grid-cli/internal/cmd/login.go

```diff
@@ -53,3 +53,13 @@ func Login() error {
 	log.Info().Msg("configuration saved")
 	return nil
 }
+
+// validate hex format of mnemonics
+func isHexValid(mnemonics string) bool {
```

Comment:
I think you can use `subkey.DecodeHex()`
- https://pkg.go.dev/github.com/vedhavyas/go-subkey#DecodeHex